packages/elements-react/src/theme/default/utils/url.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2024 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+export function restartFlowUrl(
+  flow: { request_url?: string; return_to?: string },
+  fallback: string,
+) {
+  return flow.request_url || appendReturnTo(fallback, flow.return_to)
```

Comment:
Why does this use the `request_url`? This URL is the callback URL for OIDC in account linking flows.

E.g. the user clicks on the "back button" in the flow, and is redirected back to the OIDC callback which leads to 404 (or similar).

Reply:
It‘s needed for all other flows and probably needs a fix in kratos account linking

Reply:
What is it needed for?

Reply:
It's needed to restart any flow (back button functionality) and properly set return to urls, requested aal, refresh, and other query parameters